### DISCUSSION TITLE
glTF Working Group Meeting 2021-12-23

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#43] [General]
# Agenda
- Brainstorming on 2022 in the glTF SubGroup
- Discuss https://github.com/omigroup/gltf-extensions/issues/42

**Comment by robertlong** [comment]
# Goals for 2022
- Ratify OMI_audio_emitter extension
- VRM/glTF workflow retargeting animations etc.
- More extensions!
  -  Suggest new ones here https://github.com/omigroup/gltf-extensions/discussions/7

# GLTF Animations as an Asset
- @eviemarie outlined workflows after the last meeting: https://github.com/omigroup/gltf-extensions/issues/42
- Created https://github.com/omigroup/gltf-extensions/issues/44
- @madjin's notes here: https://hackmd.io/@XR/mocap-interop
- Need for a Blender 3.0 .blend file template skeleton
- Four potential projects to collaborate with for web:
  - Webaverse
    - https://github.com/webaverse/app/blob/master/avatars/avatars.js
    - https://github.com/webaverse/sword/blob/f6777403e2f883f985234ff5a8d1dd28a06f2e01/.metaversefile#L16
  - Ossos
    - https://github.com/sketchpunklabs/ossos
  - JanusXR
    - https://github.com/jbaicoianu/janusweb/blob/master/scripts/janusghost.js#L139
    - https://github.com/jbaicoianu/janusweb/blob/5ff6926a3e12fc08a787f028ee1b03fb913007cb/scripts/janusplayer.js
  - XREngine
    - https://github.com/XRFoundation/XREngine/blob/dev/packages/engine/src/avatar/SkeletonUtils.ts  
  
  # Note
  
  No meeting next week! See you all in 2022 👋 🥳

**Comment by madjin** [comment]
How Webaverse does animation interop:

the sword defines the animations to use: https://github.com/webaverse/sword/blob/f6777403e2f883f985234ff5a8d1dd28a06f2e01/.metaversefile#L16
the animation blend comes from the avatar system
and webaverse retargets it onto the VRM
